src/dsql/DdlNodes.h

```diff
@@ -1790,6 +1790,7 @@ class CreateIndexNode : public DdlNode
 	MetaName name;
 	bool unique;
 	bool descending;
+	bool active;
```

Comment:
Looks like this field is not initialized.

Reply:
It is unconditionally assigned in parse.y.

Reply:
Assigning value twice in row is just a waste of time.

Reply:
Do you still wonder why most of your PRs nobody wants to review and/or accept ?

Reply:
If you have a testcase that shows a problem - share it.